cli/rt/11_workers.js

```diff
@@ -39,20 +39,6 @@
   const encoder = new TextEncoder();
   const decoder = new TextDecoder();
 
-  class MessageEvent extends Event {
-    constructor(type, eventInitDict) {
-      super(type, {
-        bubbles: eventInitDict?.bubbles ?? false,
-        cancelable: eventInitDict?.cancelable ?? false,
-        composed: eventInitDict?.composed ?? false,
-      });
-
-      this.data = eventInitDict?.data ?? null;
-      this.origin = eventInitDict?.origin ?? "";
-      this.lastEventId = eventInitDict?.lastEventId ?? "";
-    }
-  }
```

Comment:
Does it make sense to move the MessageEvent in a separate PR?

Reply:
i could if you want, but i dont see all that much point to it